internal/pkg/cli/svc_show.go

```diff
@@ -172,7 +172,7 @@ func (o *showSvcOpts) askSvcName() error {
 	}
 
 	if len(svcNames) == 0 {
-		log.Infof("No services found in application %s\n.", color.HighlightUserInput(o.AppName()))
+		log.Infof("No services found in application %s\n", color.HighlightUserInput(o.AppName()))
```

Comment:
lol, should it be:
```suggestion
		log.Infof("No services found in application %s.\n", color.HighlightUserInput(o.AppName()))
```
I think we end our printed messages with a punctuation.